### PR TITLE
Cleanup repo

### PR DESCRIPTION
**Pull request by sonalranjit**
cleaned up the repo and filenames and the README

### CODE REVIEW

File: csv2Urls.py [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
+#path to csv
 csv_Path = 'eis-listing.csv'
+
+#Base url
 base_HTML_EPA_url = 'https://cdxnodengn.epa.gov/cdx-enepa-II/public/action/eis/details?eisId='
 
 with open(csv_Path) as csvfile:
     reader = csv.DictReader(csvfile)
-    text_file = open("EISIds.txt", "w")
+    text_file = open("EIS_URLs.txt", "w")
```
Comment (dcwalk) [inline csv2Urls.py]: Maybe we uncouple the code from EPA/EIS? And we call this "paths.txt" to be inline with our other tools?

File: csv2Urls.py [hunk 2]
```diff
@@ -1,13 +1,16 @@
 import csv
 
+#path to csv
 csv_Path = 'eis-listing.csv'
```
Comment (dcwalk) [inline csv2Urls.py]: Can we move this to a variable or something? again just thinking about abstracting out the EIS/EPA stuff

File: csv2Urls.py [hunk 3]
```diff
@@ -1,13 +1,16 @@
...
+
+#Base url
 base_HTML_EPA_url = 'https://cdxnodengn.epa.gov/cdx-enepa-II/public/action/eis/details?eisId='
```
Comment (dcwalk) [inline csv2Urls.py]: I think we could use "url.txt" with the other tools to cover a base url?

**Comment by sonalranjit** [comment]
@dcwalk ive updated the code to be uncoupled, can merge you approve

**Review by dcwalk — APPROVED** [review]

**Comment by dcwalk** [comment]
Nice!

**Pull request merged**